packages/build-utils/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@metamask/build-utils",
+  "version": "0.0.0",
+  "description": "Utilities for building MetaMask applications",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/build-utils#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/build-utils",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest --reporters=jest-silent-reporter",
+    "test:clean": "jest --clearCache",
+    "test:verbose": "jest --verbose",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/utils": "^8.2.0"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.4.3",
+    "@types/eslint": "^8.44.7",
+    "@types/jest": "^27.4.1",
+    "deepmerge": "^4.2.2",
+    "eslint": "^8.44.0",
```

Comment:
I see that this is in `devDependencies`, but `lintTransformedFile` takes an argument that's of type `ESLint`. So, two questions:

1. Where does this type come from — does it come from this package or `@types/eslint`? Do we need both or just one?
2. `ESLint` will show up in the type definition file for `src/transforms/utils.ts`, so anyone who uses this library in a TypeScript will need that type in order to compile their code. We've encountered problems with this type of thing before and the solution has been to move the library in question to `dependencies`. What are your thoughts on moving either `eslint` or `@types/eslint` to `dependencies`?